functions/helloworld/index.js

```diff
@@ -32,6 +31,7 @@ functions.http('helloGET', (req, res) => {
 // [END functions_helloworld_get]
```

Comment:
it looks like the region tag for functions_helloworld_get is just the functions-framework declaration. is that intentional?

Reply:
It also includes a sample at (post-change) lines 28-30.

(post-change) lines 22 and 24 are different region tags.